src/main.js

```diff
@@ -15,7 +15,7 @@ Vue.use(IconsPlugin)
 import 'bootstrap/dist/css/bootstrap.css'
 import 'bootstrap-vue/dist/bootstrap-vue.css'
 
-Vue.config.productionTip = false
+Vue.config.productionTip = process.env.NODE_ENV !== "production";
```

Comment:
actually for this, I'd have to see what heroku environment variables we have.  if it's something other than 'production' for NODE_ENV, I might have to change this.  but I don't have access to the heroku project

Reply:
We're not currently setting a var for that. I can create one though

Reply:
can you check if heroku is setting that for us? looks like it's been that way for most node related applications for a while, but I'd like to verify

https://devcenter.heroku.com/changelog-items/688

Reply:
It's not in the config vars, but we can console log it to test

![image](https://user-images.githubusercontent.com/37924/95474351-3c5ab080-0942-11eb-81fb-d3772a49ed07.png)